lib/util/LocationStatusManager.js

```diff
@@ -507,11 +516,23 @@ class LocationStatusManager {
                 return next();
             }
             const paused = this._locationStatusStore[location].getServicePauseStatus(service);
-                if (!paused) {
+            if (!paused) {
+                this._updateServiceStatusForLocation(location, err => {
+                    if (err) {
+                        this._logger.debug('failed to pause service', {
```

Comment:
maybe this log should be moved to `_updateServiceStatusForLocation`, may be less redundant than having it after each call...